packages/core/admin/admin/src/content-manager/components/EditViewDataManagerProvider/utils/cleanData.js

```diff
@@ -20,10 +23,12 @@ const cleanData = ({ browserState, serverState }, currentSchema, componentsSchem
    * @param {object} browserState – the modifiedData from REDUX
    * @param {object} serverState – the initialData from REDUX
    * @param {*} schema
+   * @param {string} path
```

Comment:
Should we add the correct JSDOC for `pathToParent` ?

Reply:
Ah i didn't change the name, yep good suggestion. I'll make the change.